service/templates/distros.html

```diff
@@ -517,6 +517,18 @@ <h2 style="display:inline-block; padding-right:15px">Configure
           <div class="icon fa fa-warning distro-error" ng-show="form.userName.$dirty && form.userName.$error.required || form.userName.$invalid">SSH
             user is required</div><br>
         </div>
+        <div class="dropdown">
+          <span class="distro-menu-title">Bootstrap Method:</span>
+            <button class="btn btn-default dropdown-toggle" type="button" data-toggle="dropdown" aria-expanded="true" ng-disabled="readOnly">
```

Comment:
Why not use a select for this?

Reply:
If I use select, the formatting won't match the rest of the page. Some of the containers options also use select, but they have a completely different style from the rest of the page.